overlord/snapstate/autorefresh.go

```diff
@@ -126,6 +128,30 @@ func (m *autoRefresh) clearRefreshHold() {
 	tr.Commit()
 }
 
+// AtSeed configures refresh policies at end of seeding.
+func (m *autoRefresh) AtSeed() error {
+	// on classic hold refreshes for up to ~6h after seeding
+	if release.OnClassic {
+		var t1 time.Time
+		tr := config.NewTransaction(m.state)
+		err := tr.Get("core", "refresh.hold", &t1)
+		if !config.IsNoOption(err) {
+			// already set or error
+			return err
+		}
+		// the 20mins tries to ensure that an autoRefresh.Ensure
+		// recalculating the refresh happens before the full 6h
+		// (default Ensure interval is 5mins)
+		maxDelta := 6*time.Hour - 20*time.Minute
+		delta := time.Duration(rand.Int63n(int64(maxDelta)))
```

Comment:
I thought we were going to delay for 6h, but this is random on 0-6h, which means we may not delay at all too. What am I missing?